ztp/ran-crd/site-config-crd.yaml

```diff
@@ -102,6 +102,9 @@ spec:
                           - du
                           - cu
                           - cu-du
+                      networkType:
```

Comment:
it will better to have this as enum values OpenShiftSDN & OVNKubernetes with default value.

Reply:
Changed the CRD with enum and default vlaues.